odysseus/convertFtbQuests.ts

```diff
@@ -475,10 +476,16 @@ export const convertFtbQuests = async (input: QuestInputFileSystem, output: Ques
                 const questTitle = quest.title ? formatString(quest.title) : inferredData?.title;
                 const questSubtitle = quest.subtitle ? formatString(quest.subtitle) : undefined;
                 const questIcon = quest.icon ? convertIcon(quest.icon) : inferredData?.icon;
+                let hidden: "LOCKED" | "IN_PROGRESS" | "COMPLETED" | "COMPLETED_CLAIMED" | undefined = undefined;
+                if (quest.invisible) {
```

Comment:
Minor nitpick, but I think it'd look better if there was a newline between the variable declaration and if statement.
If not, that's fine. I'll approve anyways since I haven't setup a formatter for this.

Reply:
too quick ee2194d